index.html

```diff
@@ -48,7 +48,7 @@ <h1>Tic Tac Toe</h1>
 </table>
 
 <h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
-<h3 hidden>Try again...</h3>
+<h3 hidden>Try again... ❌</h3>
```

Comment:
```suggestion
<h3 hidden>Try again... 🔁</h3>
```

how about a repeat icon?